x-pack/plugins/maps/public/components/tooltip_selector/tooltip_selector.tsx

```diff
@@ -156,7 +158,18 @@ export class TooltipSelector extends Component<Props, State> {
 
   _renderProperties() {
     if (!this.state.selectedFieldProps.length) {
-      return null;
+      return (
+        <EuiText size="s" textAlign="center">
+          <p>
+            <EuiTextColor color="subdued">
+              <FormattedMessage
+                id="xpack.maps.tooltipSelector.emptyState.description"
+                defaultMessage="Add a tooltip field to view and create filters from field values."
```

Comment:
fwiw, `Add a tooltip field to create filters from field values.` works just as well ? 🤷